examples/chef/chef_config.py

```diff
@@ -0,0 +1,17 @@
+#
+# USER DEFINED INFORMATION
+#
+configured = False
+#configured = True
+
+# NRFConnect
+
+nrfconnectFolder = "/opt/nordic/ncs/v1.9.1"
+nrfConnectSerialDevice = "/dev/tty.usbmodem0*"  # Typical MacOS path
+# nrfConnectSerialDevice = "/dev/ttyACM0" # Typical Linux path
```

Comment:
this is python - could we check `sys.platform` or something like that?

Reply:
Can be done as a followup: I see that the code asks for `configured=True` to be done.

Generally I believe this has to be fixed: dirtying the tree should not be required to build/configure things. At most, we can ask for a git-ignore file for settings (e.g. sdkconfig does this) and have python scripts as helpers here.

Also folders for NRFConnect and similar can be defaulted from OS environment variables, like our build scripts do.